### DISCUSSION TITLE
atLevel version 1.3 not working

### DISCUSSION THREAD

**Discussion by sysmat** [qos-ch/logback#592] [General]
- slf4j 2.0.0, logback 1.3
- in logback.xml I have INFO level, but when using in java as DEBUG I can still log message in file

```java
LOG.atLevel(Level.DEBUG)
            .setMessage(() -> "user={}")
            .addArgument(() -> user)
            .log();
```

**Comment by ceki** [comment]
Thanks for the heads up. This is an SLF4J issue. See https://jira.qos.ch/browse/SLF4J-559

**Reply by sysmat** [reply]
- with **slf4j 2.0.1**, logback **1.3.1** it works as expected, trhx
